SeriesMaster-Frontend/App/api/api.js

```diff
@@ -0,0 +1,3 @@
+export default {
+  server: "http://192.168.178.44:3001",
```

Comment:
If someone else likes to develop with you, it might be useful to allow to change the address.
Maybe you can read from environment variables here?

Reply:
Great, thanks!